packages/in_app_purchase/in_app_purchase_android/README.md

```diff
@@ -11,6 +11,12 @@ so you do not need to add it to your `pubspec.yaml`.
 However, if you `import` this package to use any of its APIs directly, you
 should [add it to your `pubspec.yaml` as usual][3].
 
+## Alternative/UserChoice Billing
+
+Alternative and UserChoice billing from google play is exposed from this package. Using this feature requires google play app configuration, checking if the feature is available (`isAlternativeBillingOnlyAvailable`) and informing users that google play does not handle all aspects of purchase (`showAlternativeBillingOnlyInformationDialog`). After those calls then you can call `setBillingChoice` and respond when a user attempts a purchase.
```

Comment:
Capitalize "Google Play" everywhere 

also 
```suggestion
Alternative and UserChoice billing from google play are exposed from this package. Using this feature requires google play app configuration, checking if the feature is available (`isAlternativeBillingOnlyAvailable`) and informing users that google play does not handle all aspects of purchase (`showAlternativeBillingOnlyInformationDialog`). After those calls then you can call `setBillingChoice` and respond when a user attempts a purchase.
```

Also, do you need to call `isAlternativeBillingOnlyAvailable` to use UserChoice billing?

Reply:
It looks like if you setup user choice billing the sdk does the check for you "The system checks if the user's [Google Play country](https://support.google.com/googleplay/answer/7431675) is a country that supports alternative billing with user choice (i.e. a supported country)." https://developer.android.com/google/play/billing/alternative/alternative-billing-with-user-choice-in-app

Reply:
I will make it clear that most of this section is about alternative billing only.